packages/frontend/component/src/ui/date-picker/date-picker.tsx

```diff
@@ -74,6 +75,10 @@ const HeaderLayout = ({
 export const AFFiNEDatePicker = ({
   value,
   onChange,
+  onSelect,
+
```

Comment:
No need spaces？

Reply:
For some situation in journal that some date is selected, but user can switch to another page (not a journal page), the date should still be active, and when click date again, should trigger 'change journal date'.

`onChange` won't triggered when click a date that is already active.